### PR TITLE
bug: Accesibility fixes and remove a code warning

### PR DESCRIPTION
**Pull request by jorelosorio**
Closes https://github.com/AtB-AS/kundevendt/issues/3857
Depend on: https://github.com/AtB-AS/firestore-configuration/pull/171

### CODE REVIEW

**Review by gorandalum — CHANGES_REQUESTED** [review]
Shouldn't use index as key:
https://react.dev/learn/rendering-lists#why-does-react-need-keys

File: src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/CityZoneMessage.tsx [hunk 1]
```diff
@@ -49,10 +50,13 @@ export const CityZoneMessage: React.FC<CityZoneMessageProps> = ({
   };
 
   const messageActions = selectedCityZones.map((cityZone) => ({
+    id: `${cityZone.id}_action`,
     text: cityZone.name,
     onPress: () => openUrlForCityZone(cityZone),
   }));
 
+  console.log(messageActions);
```
Comment (gorandalum) [inline src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/CityZoneMessage.tsx:58]: Should be removed

**Review by gorandalum — APPROVED** [review]
Looks good! Just a note that I think using `name` as key instead of adding `id` would work just as well.

File: src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/CityZoneMessage.tsx (hunk 1 above)
Comment (jorelosorio) [reply src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/CityZoneMessage.tsx]: Oops! My fault, removed now ;)

**Comment by jorelosorio** [comment]
> Looks good! Just a note that I think using `name` as key instead of adding `id` would work just as well.

I thought the same, but the name might not be OK for an ID because it might contain special characters, spaces, etc. So it is nice to have a proper field for this.

**Comment by gorandalum** [comment]
> I thought the same, but the name might not be OK for an ID because it might contain special characters, spaces, etc. So it is nice to have a proper field for this.

Just FYI React keys doesn't have limitations on what they can include, and can have spaces, special characters etc. The only rule is that they should be unique.

**Pull request merged**